widget/embedded/src/utils/colors.ts

```diff
@@ -0,0 +1,86 @@
+import type { Colors, WidgetColors } from '../types';
+
+const Range = 255;
+const Sixteen = 16;
+const Col = 3;
+const NumberOfColors = 10;
+const Five = 5;
+const Suffix = 100;
+const ThirtyTwo = 32;
```

Comment:
Please move these variables to functions if they are only used in one of the functions and use meaningful names insteaf of `Five`, `Sixteen`, `ThirtyTwo`, ... 
Also please use one of these conventions for naming: `NAME` or` name`.

Reply:
fixed